lib/bolt_ext/server-config.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'hocon'
+
+class TransportConfig
+  attr_accessor :port
+
+  def initialize(global = nil, local = nil)
+    # TODO: is 8144 a reasonable default port? It doesn't seem to be in use per
+    # https://confluence.puppetlabs.com/display/ENG/Choosing+a+port+for+services,
+    # and is close to other orch/pcp service ports
+    @port = 8144
+
+    global_path = global || '/etc/puppetlabs/bolt-server/conf.d/bolt-server.conf'
+    local_path = local || File.join(ENV['HOME'].to_s, ".puppetlabs", "bolt-server.conf")
```

Comment:
HOME may not always be set.

Reply:
I guess failure is acceptable in that case.

Reply:
Should it fail, or warn and just load the global path?

Reply:
warn and just load global path